dask/dataframe/tests/test_dataframe.py

```diff
@@ -5108,6 +5108,15 @@ def test_index_is_monotonic_dt64():
     assert_eq(s_2.index.is_monotonic_decreasing, ds_2.index.is_monotonic_decreasing)
 
 
+def test_is_monotonic_empty_partitions():
+    df = pd.DataFrame({"a": [1, 2, 3, 4], "b": [4, 3, 2, 1]})
+    ddf = dd.from_pandas(df, npartitions=2)
+    assert ddf[ddf["a"] >= 3]["a"].is_monotonic.compute()
+    assert ddf[ddf["a"] >= 3].index.is_monotonic.compute()
+    assert not ddf[ddf["a"] >= 3]["a"].is_monotonic_decreasing.compute()
+    assert not ddf[ddf["a"] >= 3].index.is_monotonic_decreasing.compute()
```

Comment:
Minor point, but it would be good to use `assert_eq` to make sure that the Dask version works, but it also matches the equivalent pandas result